clients/web/test/spec/userSpec.js

```diff
@@ -96,6 +96,16 @@ describe('Create an admin and non-admin user', function () {
         }, 'front page to display');
     });
 
+    it('check the user count with an admin user', function () {
+        girderTest.login('admin', 'Admin', 'Admin', 'adminpassword!')();
+        girderTest.goToUsersPage()();
+        runs(function () {
```

Comment:
That's reasonable. You can still reduce some redundancy though, if you let this spec run first and:
* login
* go to users page
* check that the count is correct

Then have the `'check the user count with an admin user'` spec, which will:
* logout
* wait for the front page to display

You do have some state carrying over between specs (specifically, the fact that there's a logged in user, navigated to the users page), which isn't ideal, but is more than good enough for Girder front-end tests.